### PR TITLE
some minor improvements

### PR DESCRIPTION
**Pull request by Gemorroj**
drop unused imports, unused variables, `dirname(__FILE__)` ->` __DIR__`, cs fixes, ...

### CODE REVIEW

File: pgo/cases/joomla/TrainingCaseHandler.php [hunk 1]
```diff
@@ -48,8 +48,6 @@ protected function getToolFn() : string
 
 	protected function setupDist() : void
 	{
-		$port = $this->getHttpPort();
-		$host = $this->getHttpHost();
```
Comment (weltling) [inline pgo/cases/joomla/TrainingCaseHandler.php]: Please keep these two calls where ever in the `TrainingCaseHandler` classes. The `TrainingCaseHandler::init()` needs to setup the server config, these calls will generate appropriate config items if not present. Some cases like Wordpress use a cli tool which needs this info, too. Cases like Symfony the tool doesn't need this however. Please check carefully to not to remove method calls as that might lead to bugs.

Thanks.

**Comment by weltling** [comment]
Seems OK at first glance. Please also check the note.

Thanks.

File: pgo/cases/joomla/TrainingCaseHandler.php (hunk 1 above)
Comment (Gemorroj) [reply pgo/cases/joomla/TrainingCaseHandler.php]: Thank you, indeed, I did not notice. Can be rename these methods? Something like `getOrMakeHttpPort`/`getOrInitHttpPort`.

**Comment by weltling** [comment]
Merged as 1b8473489fb61cb2f8402ab3599c0de146ad0b31. Thanks!

File: pgo/cases/joomla/TrainingCaseHandler.php (hunk 1 above)
Comment (weltling) [reply pgo/cases/joomla/TrainingCaseHandler.php]: Could rename, yep. Not critical. 

Thanks.

**Pull request closed without merging**